scripts/performance/perf_load/perf_gen_req_parser.py

```diff
@@ -14,7 +15,7 @@ class ReqTypeParser:
          "revoc_reg_entry": RGEntryRevoc, "get_nym": RGGetNym, "get_attrib": RGGetAttrib, "get_schema": RGGetSchema,
          "get_cred_def": RGGetDefinition, "get_revoc_reg_def": RGGetDefRevoc, "get_revoc_reg": RGGetEntryRevoc,
          "get_revoc_reg_delta": RGGetRevocRegDelta, "get_payment_sources": RGGetPaymentSources, "payment": RGPayment,
-         "verify_payment": RGVerifyPayment}
+         "verify_payment": RGVerifyPayment, "config": RGConfig}
```

Comment:
Name "config" is too general. Rename it to something like "pool_cfg_writes" and RGConfig rename accordingly.